src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -168,6 +168,11 @@ def validate_client_parameters(cmd, namespace):
     if n.sas_token:
         n.sas_token = n.sas_token.lstrip('?')
 
+    # account name with secondary
+    if n.account_name.endswith('-secondary'):
```

Comment:
How does user know this trick?

Reply:
specify --account-name `xxxx-secondary` 
here is request url:
urllib3.connectionpool : Starting new HTTPS connection (3): zuhlrs-secondary.blob.core.windows.net:443